### ISSUE TITLE
Illegal character confusion/bad ux

### ISSUE THREAD

**Issue by l300lvl** [ganeshrvel/openmtp#365]
Not necessarily a bug but this one had me stumped several times. To reproduce this:

Try to copy any file with other files as a group transfer with a quote symbol in the filename of any of the files and the transfer dialog simply pauses and never completes, with no visible warning or error. It just seems to hang eternally. 

The way I did detect what is happening is by copying a single file with the bad character, when transferring only one file an illegal character notice is indeed given. When transferring multiple files some good/some with the illegal char, no notice ever came up.

To be sure, I renamed the file after getting the notice, and it indeed copied.

What really really boggles my mind is: if I compress a file containing the _illegal_ character and transfer the zip and extract it under the phone, the character exists just fine? Is it really so illegal?

**Comment by gotjoshua** [comment]
i agree that this is a bit strange ux... i had files with a "/" character (from the audible title that libation created)
after a bulk rename the files all transferred.

The worst part of it was that it copied the folder, but then it was empty and there was no error statement.
As the OP mentions, if i try to copy a single file, or even all the files that are within the folder, i get an error banner.

Maybe this is an easy fix, as it seems the error is just being swallowed for a folder.
